HarpGuru/OverlayMenu/Menus/CovariantMenu/components/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import {MenuContainer} from '../../MenuContainer'
+import type {MenuProps} from '../../../types'
+import { DisplayModeOption, HarpKeyOption, RootPitchOption, PozitionOption } from '../../../MenuOptions'
+
+
+export const CovariantMenu = (props: MenuProps): React.ReactElement => {
+  return (
+    <MenuContainer>
+      <HarpKeyOption {...props} />
+      <PozitionOption {...props} />
+      <RootPitchOption {...props} />
+      <DisplayModeOption {...props}/>
+    </MenuContainer>
+  )
+}
```

Comment:
This feature should probably be renamed `CovarianceMenu`

Reply:
No it shouldn't.
1: It is a menu of Covariants, so perhaps CovariantsMenu would be better.
2: This is not a significantly important distinction for this to matter. I'm not even sure which is better.